src/components/v5/common/ActionSidebar/hooks/permissions/useHasNoDecisionMethods.ts

```diff
@@ -33,6 +38,15 @@ const useHasNoDecisionMethods = () => {
     return false;
   }
 
+  // Check if the user has the required permissions in root domain
+  // for action types which can only be actioned in root domain
+  const userRootRoles = getUserRolesForDomain(colony, user.walletAddress, 1);
+  if (!requiredPermissions.every((role) => userRootRoles.includes(role))) {
+    if (actionType === Action.ManageVerifiedMembers) {
+      return true;
+    }
+  }
+
```

Comment:
Would it be possible to somehow reuse `getPermissionsDomainIdForAction` helper here to determine which domain you need to check permissions in? It would be great to have those rules defined in one place